.github/workflows/playwright.yml

```
@@ -50,7 +50,7 @@ jobs:
       matrix:
         testSuite: [
           'ai',
-          'onBoarding',
+          #'onBoarding',
```

```suggestion
          'onBoarding',
```
Can we bring this back?